### ISSUE TITLE
Run dtw error with MemoryError.

### ISSUE THREAD

**Issue by leijue222** [statefb/dtwalign#7]
Loading two wav files A and B.
Using A to align B with DTW, and save the aligned audio file.
Like you [example](https://htmlpreview.github.io/?https://github.com/statefb/dtwalign/blob/master/example/example.html) shows: `warp one to the other`

But I failed when I using `dtw(x,y)`.

### #example test data shape
```
x = np.sin(2*np.pi*3.1*np.linspace(0,1,101)), x += np.random.rand(x.size)
y = np.sin(2*np.pi*3*np.linspace(0,1,120)), y += np.random.rand(y.size)

# x.shape:  (101,) y.shape:  (120,)
```
### # load wavs data shape
```

x, sr1 = librosa.load('wavs/yifan.wav')
y, sr2 = librosa.load('wavs/baker.wav')

# x.shape:  (115200,) y.shape:  (119808,)
```

error:

```
  File "/media/yiwei/600G/tts/voice_conversion/dtw/test_dtwalign.py", line 32, in <module>
    res = dtw(x, y)
  File "/home/yiwei/anaconda3/envs/chat/lib/python3.6/site-packages/dtwalign/dtw.py", line 87, in dtw
    X = cdist(x, y, metric=dist)
  File "/home/yiwei/anaconda3/envs/chat/lib/python3.6/site-packages/scipy/spatial/distance.py", line 2720, in cdist
    dm = np.empty((mA, mB), dtype=np.double)
MemoryError
```

**I can solve the error with y=y[0:a small digit] or x=x[0:a small digit]**

Both A and B are **only 6 seconds**, there is no reason for 6 seconds of audio to take up so much memory. I still have a few minutes of audio to align. . .
Could you help me to solve the error?

**Comment by statefb** [comment]
DTW requires distance matrix with size of A * B i.e. 115200 * 119808 in your case.  
If it's difficult to prepare more memory, might want to re-sample signal with lower frequency.

**Comment by leijue222** [comment]
Thanks for your reply.
I tried [pierre-rouanet/dtw](https://github.com/pierre-rouanet/dtw)  implementation, it's ok to run 70-second audio signals, but the speed is too low.
But he lacks the realization of the `warp one to the other` step, so I prefer your [dtwalign](https://github.com/statefb/dtwalign) implementation.

After your comments, I resampled the audio signal from 22050 to 16K, 16K HZ is a small sample rate, but I still failed...
My CPU has 32G of memory, and It is unreasonable to process 6-second audio signals...

Is there any other way to solve it?
